packages/vm/README.md

```diff
@@ -23,8 +23,8 @@ To rebuild the test contracts, go to the repo root and do
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_hackatom",target=/code/contracts/hackatom/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.9.1 ./contracts/hackatom \
-  && cp contracts/hackatom/contract.wasm packages/vm/testdata/contract_0.11.wasm
+  cosmwasm/rust-optimizer:0.10.3 ./contracts/hackatom \
+  && cp artifacts/hackatom.wasm packages/vm/testdata/contract_0.11.wasm
```

Comment:
Let's stick with `cosmwasm/rust-optimizer:0.9.1` for now, which is fine for 0.11 builds of a single contract. We can than upgrade rust-optimizer in all places of this repo separatly: https://github.com/CosmWasm/cosmwasm/issues/562